src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -16,18 +16,28 @@ namespace Microsoft.Authentication.MSALWrapper.AuthFlow
     /// </summary>
     public class AuthFlowExecutor
     {
+        /// <summary>
+        /// The amount of time to wait before we start warning on stderr about waiting for auth.
+        /// </summary>
+        public static TimeSpan WarningInterval = TimeSpan.FromSeconds(10);

```

Comment:
I would not call this an `interval` because interval implies repetition. Consider `WarningDelay`? We only wait for this one time, before using the actual 30 sec interval below.